__tests__/services/fhir.spec.ts

```diff
@@ -0,0 +1,305 @@
+import {
+    getFHIRResource,
+    getFHIRResources,
+    findFHIRResource,
+    saveFHIRResource,
+    saveFHIRResources,
+    patchFHIRResource,
+    forceDeleteFHIRResource,
+    deleteFHIRResource,
+    getReference,
+    makeReference,
+    isReference,
+    getIncludedResource,
+    getIncludedResources,
+    getConcepts,
+} from '../../services/fhir';
+
+import { service } from '../../services/service';
+import { success, failure } from '../../libs/remoteData';
+
+jest.mock('../../services/service', () => {
+    return { service: jest.fn(() => Promise.resolve(success('data'))) };
+});
+
+describe('Service `fhir`', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    test('method `getFHIRResource`', async () => {
+        const reference = {
+            id: '1',
+            resourceType: 'type',
```

Comment:
use 'Patient' as universal resourceType. This resource has no mandatory fields and very good for testing purposes.